legion/engine/physics/physicsconstants.hpp

```diff
@@ -6,7 +6,7 @@ namespace legion::physics::constants
 {
     static math::vec3 gravity = math::vec3(0, -9.81, 0);
 
-    static constexpr int contactSolverIterationCount = 6;
+    static constexpr int contactSolverIterationCount = 9;
 
     static constexpr int frictionSolverIterationCount = 4;
```

Comment:
I think generally things like counts and sizes are better kept as size_type instead of integers, even if not only for semantics, then for the promise that it won't be negative.